### DISCUSSION TITLE
命名规范不显示相关内容

### DISCUSSION THREAD

**Discussion by bonghoxxi** [metatube-community/jellyfin-plugin-metatube#122] [General]
下载了插件，着实是进入了新世界。
不过“命名规范”栏目不显示相关内容，使得一些片子不该如何命名..
望大佬修复下..

同时，很想向大佬聊表一下心意，估计这样的人也不少，望大佬能告知一些渠道～

新年快乐！

**Comment by KuanKan410** [comment]
顶一顶，目前还是不显示命名规范内的内容，萌新想看看命名规范整理一下影片

**Comment by Felix2yu** [comment]
https://raw.githubusercontent.com/metatube-community/metatube-community.github.io/main/docs/wiki/naming-rules.md
都在这里
